# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pmud/X11R7-lib64.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pmud-0.10/Makefile.vanilla	2006-03-29 08:28:46.000000000 +0200
+++ pmud-0.10/Makefile	2006-03-29 08:29:17.000000000 +0200
@@ -39,7 +39,7 @@
 	$(CC) $(CFLAGS) -o fblevel fblevel.c
 
 xmouse: xmouse.c
-	$(CC) $(CLAGS) -o xmouse xmouse.c -L/usr/X11R6/lib -lX11
+	$(CC) $(CLAGS) -o xmouse xmouse.c  `pkg-config x11 --libs`
 
 tcp.o: tcp.c tcp.h
 	$(CC) $(CFLAGS) -c tcp.c
